src/coreclr/jit/codegencommon.cpp

```diff
@@ -12551,6 +12551,17 @@ void CodeGen::genPoisonFrame(regMaskTP regLiveIn)
 
         assert(varDsc->lvOnFrame);
 
+        int size = (int)compiler->lvaLclSize(varNum);
+
+        if (size / TARGET_POINTER_SIZE > 16)
```

Comment:
Nit -- always like to see these full parenthesized:
```suggestion
        if ((size / TARGET_POINTER_SIZE) > 16)
```